app/CPReflectionService/RBObject+CocoaPods.rb

```diff
@@ -62,7 +62,7 @@ def self.analyze_podfile(podfile, installation_root)
     end
 
     def self.all_pods
-      Pod::SourcesManager.aggregate.all_pods
+      Pod::Source::Manager.aggregate.all_pods
```

Comment:
this will crash -- I'd use `Pod::Config.instance.sources_manager.aggregate.all_pods`